3 - Assignment/240604 - Rust Basic + Roguelike Game, Week 5/luhn-algorithm/src/lib.rs

```diff
@@ -16,12 +18,18 @@ pub fn luhn(cc_number: &str) -> bool {
             }
 
             double = !double;
-        } else {
+        } else if c == ' ' {
```

Comment:
빈 칸인지 확인하는 메소드가 있습니다.
```suggestion
        } else if c.is_whitespace() {
```